### DISCUSSION TITLE
Best Way To PR for this Repository

### DISCUSSION THREAD

**Discussion by roni-estein** [wire-elements/modal#105] [General]
Hey @PhiloNL,

Thanks for the great package. Haven't seen any movement on [the flickering issue #26](https://github.com/wire-elements/modal/issues/26) but I've noticed you've been putting out great content and really cool projects, so thanks for those as well. There is a note that it's being worked on, but if you have some ideas and some direction I'd be happy to put together a PR for review if it would take something off your plate. I know how daunting it can be looking at a pile of items at work and then seeing other people adding more items on your generous open source. But we are all happy to lend a hand or a few hours if you are interested.

If you have the time, in general a guide of what you would like to see in a PR for this repository would help us craft PR's that look more like what you are expecting. And for this specific example, if you already have a solution you think would work, maybe a quick response with an outline of that too.

Thanks for all you do :)

**Comment by PhiloNL** [comment]
Hey @roni-estein 

Thanks for the nudge. I haven't taken the time yet to dive into this issue.
The lookup table seems like a good solution like you mentioned [here](https://github.com/wire-elements/modal/issues/26). I don't think this will introduce any breaking changes so that would be perfect. If you have the time to create a PR I would really appreciate it 🙏

**Reply by roni-estein** [reply]
Will do :) I'll re-pull down this project with with updated dependancies and see if there is anything different.
